### DISCUSSION TITLE
introducing myself and an initial question: marian-nmt/marian-dev vs marian-nmt/marian ?

### DISCUSSION THREAD

**Discussion by ekoontz** [marian-nmt/marian-dev#985] [Q&A] [answered]
Hi Folks,
I learned about this project through https://github.com/mozilla/firefox-translations/ and I'm interested to learn more about neural network machine translation. I know a little C++ but nothing about neural networks yet. I was able to compile marian on a AWS EC2 g5.xlarge, so that's my progress so far. My first question is about these two repos:

- https://github.com/marian-nmt/marian
- https://github.com/marian-nmt/marian-dev

These two repos look identical to me as far as their commit history - is there a reason why they are duplicated? The branches and forks are different however, with more branches on `marian-dev` - perhaps this one is for more day-to-day development, while `marian` is for releases?
Thanks for reading and looking forward to contributing in the future!
-Eugene

**Comment by snukky** [comment, marked as answer]
Hi! Yes, that's correct. We use `marian-nmt/marian-dev` for development and `marian-nmt/marian` for major releases. In most cases I would recommend using marian-dev as you get new features and bug fixes faster.

**Reply by ekoontz** [reply]
Thanks so much @snukky for your answer!
